### PR TITLE
Update README

### LINKED ISSUE #81: README instructions mistake
There is a small mistake in README.md.  Specifically where it says:

```bash
$ Where REPO_DIR points to the root of the csharp repository
cd ${REPO_DIR}/csharp/src
${GEN_DIR}/openapi/csharp.sh generated csharp.settings
```

`csharp.settings` is located in $REPO_DIR/csharp, but in the example we are in $REPO_DIR/csharp/src.  The last line of this block should instead say:

```bash
${GEN_DIR}/openapi/csharp.sh generated ../csharp.settings
```

Also:

- it might be helpful to list dependencies as: git, nodejs >= 7.10.0, docker
- it might be helpful to state that the instructions are intended to be run on a Linux system (since it's a C# library, people might assume they are Windows instructions)

### PR DESCRIPTION
**Pull request by qmfrederik**
- Document the requirement for a Linux machine with nodejs
- Fix a relative path in the instructions

Fixes #81

### CODE REVIEW

**Comment by tg123** [comment]
generator runs in docker, I think the only dependency is docker. no nodejs needed.

**Comment by qmfrederik** [comment]
Sounds good! I removed the `npm install autorest` step as well, as it seems that's not a requirement anymore.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ dotnet add package KubernetesClient
 
 ## Prerequisites
 
+You'll need a Linux machine with Docker.
```
Comment (brendandburns) [inline README.md]: it _might_ work on OS X...

**Comment by brendandburns** [comment]
LGTM, thanks.

**Pull request merged**